motion/p5js/motion-triangle/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Daniele Maselli
+author-url: https://github.com/DanieleMaselli
+title: motion-triangle
+tags: [P5.js, p5.Vector, color, triangle]
+permalink: /color/motion/
+summary: primary colors represented by a moving triangle on P5.js.
```

Comment:
P~~p~~rimary colors represented by a moving triangle in _(or using)_ ~~on~~ P5.js.